flake.nix

```diff
@@ -54,7 +54,7 @@
       cargoTOML = builtins.fromTOML (builtins.readFile ./Cargo.toml);
       cargoLock = builtins.fromTOML (builtins.readFile ./Cargo.lock);
 
-      version = "${cargoTOML.workspace.package.version}_${builtins.substring 0 8 self.lastModifiedDate}_${self.shortRev or "dirty"}";
+      inherit (cargoTOML.workspace.package) version;
```

Comment:
Does that impact `nickel --version` ?

Reply:
I don't think it does. The version mechanism in the nix build uses `self.shortRev` and gets the version number from `Cargo.toml` itself.